src/model/variable.ts

```diff
@@ -44,6 +44,10 @@ export abstract class Variable {
     get cxxrtlIdentifier(): string {
         return this.fullName.join(' ');
     }
+
+    get wcpIdentifier(): string {
+        return this.fullName.join('.');
```

Comment:
How does WCP handle variables with dots in their names? Amaranth will not escape these.

Reply:
Oh right, I guess `.` are valid in escaped identifiers. I guess we should change wcp to use the same encoding as cxxrtl to avoid problems?